.github/stale.yml

```diff
@@ -0,0 +1,61 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - internal
+  - BUG
+  - "help wanted"
```

Comment:
Any other labels that we want stalebot to not touch? Maybe `Feature`? :thinking:

Reply:
We have some sort of procedure:

https://github.com/emqx/emqx/blob/master/.github/ISSUE_TEMPLATE/assets/issue-handling.png

Updating the procedure is also welcomed!

Reply:
If I understood the diagram correctly, we only needed to add  `#triage/accepted` and `#needs-triage` as exempt tags.  

Just added them, thanks for the tip!